libs/ngx-localstorage/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "ngx-localstorage",
   "version": "5.0.0",
   "peerDependencies": {
-    "@angular/common": "^15.0.0",
-    "@angular/core": "^15.0.0"
+    "@angular/common": "^16.0.0",
+    "@angular/core": "^16.0.0"
```

Comment:
As there aren't any breaking changes in the Angular API used in this library - what do you think about setting the version to `>=15.0.0`?

Reply:
Good idea, it will leave the lib usable in Angular 17 or later if there are no changes.

Reply:
Exactly :)